packages/e2e-tests/specs/__snapshots__/adding-blocks.test.js.snap

```diff
@@ -42,7 +42,9 @@ exports[`adding blocks Should insert content using the placeholder and the regul
 <!-- /wp:quote -->
 
 <!-- wp:preformatted -->
-<pre class=\\"wp-block-preformatted\\">Pre text<br><br>Foo</pre>
+<pre class=\\"wp-block-preformatted\\">Pre text
+
```

Comment:
Would this change invalidate existing content?

Reply:
I’m not sure why it would? Only the onChange handler has changed

Reply:
Ah, right. So I expect it should remain valid for the initial load. The first edit may convert some `<br>` to newlines, which is fine.